### PR TITLE
ID-458 [Maintenance] Adds documentation for Fliplet.Widget.Namespace()

### PR DESCRIPTION
**Pull request by tonytlwu**

### CODE REVIEW

File: docs/API/core/storage.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Fliplet.App.Storage.remove('key').then(function () {
 
 ## Namespaced storage
 
-You can also create a private namespaced storage which is nor shared with the `Fliplet.App.Storage` neither with the global `Fliplet.Storage`:
+You can also create a private namespaced storage which is not shared with the `Fliplet.App.Storage` neither with the global `Fliplet.Storage`:
```
Comment (squallstar) [inline docs/API/core/storage.md:84]: `nor` is actually OK to use in this sentence but the order (nor, neither) should be swapped to neither, nor

File: docs/API/core/widget.md [hunk 2]
```diff
@@ -182,3 +182,33 @@ Fliplet.Widget.save({ foo: 1 }).then(function () {
...
+```js
+Fliplet.Foo.add(instance) // instance can be a promise but does not need to be
+```
+
+### Get an instance
+
+```js
+Fliplet.Foo.get().then((instance) => { /* Returns the first instsance */ })
+Fliplet.Foo.get('bar').then((instance) => { /* Returns the first instsance with instance.name: "bar" */ })
+Fliplet.Foo.get({ type: 'bar' }).then((instance) => { /* Returns the first instsance with instance.type: "bar" */ })
```
Comment (squallstar) [inline docs/API/core/widget.md:205]: typo across these comments on "instance"

File: docs/API/core/storage.md (hunk 1 above)
Comment (tonytlwu) [reply docs/API/core/storage.md]: Updated

File: docs/API/core/widget.md (hunk 2 above)
Comment (tonytlwu) [reply docs/API/core/widget.md]: Updated

**Review by squallstar — APPROVED** [review]

**Pull request merged**